tests/test_general_helpers.py

```diff
@@ -99,6 +100,23 @@ def test_unpad_coordinate_B01010():
     assert unpad_coordinate("B01010") == "B1010"
 
 
+# tests for pad coordinate
+def test_pad_coordinate_A1():
```

Comment:
Why not put these asserts in one function? It does not require any fixtures and is quite simplistic.

Reply:
Ok